src/components/evidence-tag.tsx

```diff
@@ -8,18 +15,45 @@ import '../styles/components/evidence-tag.scss';
 
 const size = 12;
 
-const EvidenceTag = ({ label, title, className, iconComponent, children }) => {
+type EvidenceTagProps = {
+  /**
+   * Displayed on the tag
+   */
+  label: string;
+  /**
+   * Displayed on on mouseover
+   */
+  title: string;
+  /**
+   * Decides the colour of the tag
+   */
+  className?: string;
+  /**
+   * Decides the colour of the tag
+   */
+  iconComponent: ReactElement<{ width: number; height: number }>;
+};
+
+const EvidenceTag: FC<EvidenceTagProps & HTMLAttributes<HTMLButtonElement>> = ({
+  label,
+  title,
+  className,
+  iconComponent = <EvidenceTagIcon />,
+  children,
+  ...props
+}) => {
   const idRef = useRef(v1());
   const [contentDisplay, setContentDisplay] = useState(false);
   return (
     <>
       <button
```

Comment:
Is there enough overlap with the `Button` component to be able to use it instead and avoid `evidence-tag`-specific styling?